mne/viz/_brain/tests/test_brain.py

```diff
@@ -369,6 +369,19 @@ def test_brain_save_movie(tmpdir, renderer, brain_gc):
     brain.close()
 
 
+@testing.requires_testing_data
+@pytest.mark.slowtest
```

Comment:
do not mark this `slowtest`, it will be skipped on macOS. But then I notice this problem:
```
3.15s call     mne/viz/_brain/tests/test_brain.py::test_brain_screenshot[pyvista]
1.38s setup    mne/viz/_brain/tests/test_brain.py::test_brain_screenshot[pyvista]
0.24s teardown mne/viz/_brain/tests/test_brain.py::test_brain_screenshot[pyvista]
```
it is actually quite slow, even on a system with a good renderer.

In this case we only care about the screenshot dimensions and not really any output, smoothing, etc. So rather than using `_create_testing_brain` let's just directly do it. I pushed a commit to do this but get this error:
```
mne/conftest.py:519: in brain_gc
    _assert_no_instances(Brain, 'after')
E   AssertionError: 1 after:
E   tuple: (<mne.viz._brain._brain.Brain object at 0x7fb07966ff10>, 1)
E   cell: <cell at 0x7fb04f11b520: Brain object at 0x7fb07966ff10>
E   method: <bound method Brain.toggle_interface of <mne.viz._brain._brain.Brain object at 0x7fb07966ff10>>
E   method: <bound method Brain.apply_auto_scaling of <mne.viz._brain._brain.Brain object at 0x7fb07966ff10>>
E   method: <bound method Brain.restore_user_scaling of <mne.viz._brain._brain.Brain object at 0x7fb07966ff10>>
E   method: <bound method Brain.clear_glyphs of <mne.viz._brain._brain.Brain object at 0x7fb07966ff10>>
```
Either I did something wrong with the test, or the `brain.close()` does not manage to clean things up here. Can you see what the problem is @GuillaumeFavelier ?

FYI even with this change, the test time is not great:
```
2.02s setup    mne/viz/_brain/tests/test_brain.py::test_brain_screenshot[pyvista]
```
2 sec to make a window with a single object in it is not so awesome. Any idea if this is a fundamental PyVista(Qt), vtk, or PyQt5 problem? It at least does not seem to be a fundamental PyQt5 issue, as this runs in 270 ms on my system (and 100 ms of that is the sleep I put in so I could actually see the window pop up):
```
$ time python -c "import time; from PyQt5.QtWidgets import QApplication, QWidget; app = QApplication([]); w = QWidget(); w.show(); app.processEvents(); time.sleep(0.1)"

real	0m0.272s
user	0m0.117s
sys	0m0.016s
```

Reply:
I'm not sure how pytest fixture closes the brain but using a normal function fixes the issue.

Reply:
I'll try again after 972b8a8

Reply:
I tried locally and I ran into issues when I use the fixture